cluster/manifests/z-karpenter/deployment.yaml

```diff
@@ -34,13 +34,9 @@ spec:
       securityContext:
         fsGroup: 1000
       priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
-      tolerations:
-      - key: node.kubernetes.io/role
-        value: master
-        effect: NoSchedule
       containers:
         - name: controller
-          image: "container-registry.zalando.net/teapot/karpenter-controller:v0.27.3-main-9"
+          image: "container-registry-test.zalando.net/teapot/karpenter:networkinterfaces-experimental-pr-8-3@sha256:ad8d5606be2831ce13e976c16c6333579fe3fc466e87eb7117f9c464af3b5f60
```

Comment:
registry-test :dagger:

Reply:
yes, since this is a temporary patched build to be tested only on test clusters (where the test registry is allowed)
the whole karpenter deployment is behind the feature flag which is turned off on all clusters, will be only enabled on selected test clusters

Reply:
If we intend to merge this PR then I would also suggest to rather fetch it from the production registry.

Reply:
-1 because of this